Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.yml

```diff
@@ -15,6 +15,12 @@ args:
     contains an emails contains an email). Default depth level is 3. Minimum level
     is 1, if set to 1 the script will parse only the first level email
   name: max_depth
+- required: false
+  name: forced_encoding
+  description: Use only the force encoding when parsing the message. Relevant to msg files only.
+- required: false
+  name: default_encoding
+  description: Use only the default encoding when parsing the message with the detected encoding fails. Relevant to msg files only.
```

Comment:
maybe it is good to add examples of default encodings to make it clear what strings can be added here?